src/thor/dijkstras.cc

```diff
@@ -199,7 +199,8 @@ void Dijkstras::ExpandForward(GraphReader& graphreader,
     uint32_t idx = bdedgelabels_.size();
     *es = {EdgeSet::kTemporary, idx};
     bdedgelabels_.emplace_back(pred_idx, edgeid, oppedgeid, directededge, newcost, mode_,
-                               transition_cost, path_dist, false, restriction_idx, pred.path_id());
+                               transition_cost, path_dist, false, true, restriction_idx,
```

Comment:
the changes here dont look right. you shouldnt hardcode this to `true`. you'll also want to make the changes here to look like the ones you did in bidirectional a*. if you dont do this the reachability computation will be technically wrong. speaking of which we need to think about the cheap reachability check and whether or not it should bail early and fall back ot the full check via dijkstras

Reply:
I hard-coded it here (and other unidirectional variants) since I dont want to change the existing behavior for them. This flag switched on closure pruning which is what we do currently, i.e we don't consider closed edges. Or maybe there's a gap in my understanding about how reachability works. Let me know.

Reply:
yeah so there are two things here. we have a quck reach check, where we want allowed to return false right away when an edge is closed to set the reachability low. when this happens, ie the quick reach check fails, the edge candidate gets kicked out to dijkastas to compute the actual reachability. but to do that we need to actually properly handle whether the predecessor was or wasnt closed and set pruning appropriately. the way this is here any closed edge will have low reachability and force us to search for another candidate edge. which was the case before, but when the user says hey i want to route through consecutive closures at the beginning/end of the route we need dijkstras here to actually do that.